docs/02-app/02-api-reference/04-functions/generate-metadata.mdx

```diff
@@ -521,6 +521,9 @@ export const metadata = {
 <meta property="og:image:width" content="1800" />
 <meta property="og:image:height" content="1600" />
 <meta property="og:image:alt" content="My custom alt" />
+<meta property="og:video:url" content="https://nextjs.org/video.mp4">
```

Comment:
lint is failing
```suggestion
<meta property="og:video:url" content="https://nextjs.org/video.mp4" />
```

Reply:
Oh... Thank you for check lint.

BTW, this code is html, **not JSX**.
That's why we might use plane html tag.

The real output is
`<meta property="og:video:url" content="https://nextjs.org/video.mp4">`

However, it's not big matter on this section.